workers/downloaders/tasks.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import, unicode_literals
+from celery import shared_task
+
+from bioinformatics_mill_models.models import Batch
+
+
+@shared_task
+def save_test_batch():
+    print("saving batch")
```

Comment:
I think we should also put in some Python logging statements as we grow this so that we save some output of the processes that are going on as we run this: https://docs.python.org/3/library/logging.html